dash/dash-renderer/webpack.base.config.js

```diff
@@ -31,7 +31,7 @@ const defaults = {
             {
                 test: /\.ts(x?)$/,
                 exclude: /node_modules/,
-                use: ['babel-loader', 'ts-loader'],
```

Comment:
Interesting, I wasn't aware this could be omitted. Sounds to me as though it's safer to keep it though, to keep type checking. Per https://webpack.js.org/guides/typescript/#loader:

> Note that if you're already using [babel-loader](https://github.com/babel/babel-loader) to transpile your code, you can use [@babel/preset-typescript](https://babeljs.io/docs/en/babel-preset-typescript) and let Babel handle both your JavaScript and TypeScript files instead of using an additional loader. Keep in mind that, contrary to ts-loader, the underlying [@babel/plugin-transform-typescript](https://babeljs.io/docs/en/babel-plugin-transform-typescript) plugin does not perform any type checking.